eng/common/TestResources/SubConfig-Helpers.ps1

```diff
@@ -82,7 +118,8 @@ function ShouldMarkValueAsSecret([string]$serviceName, [string]$key, [string]$va
         "TenantId",
```

Comment:
Isn't the main values we want already in this list?

Reply:
It's possible that in the storage case, the value is getting set from somewhere else to a config key that is not in this exclusion list. Was it the sub or the rg you were seeing starred out?